### ISSUE TITLE
Export Current status of joal in a text file

### ISSUE THREAD

**Issue by deadpansociety** [anthonyraymond/joal#67]
Is there a way to get information(per torrent seed/leech & up-speed) of https://raw.githubusercontent.com/anthonyraymond/joal/master/readme-assets/webui-preview.png window as a text file? 

When there's a thousand of torrents running in joal, the webui is quite unresponsive for me. Webui crashes for large sum of torrents. It'd be helpful to have some alternative

**Comment by anthonyraymond** [comment]
Not so far, but i'm going to implement a table, with pagination and search bar. It should solve the problem

**Comment by deadpansociety** [comment]
That'd be great. May you also add torrent list sorting. In particular sort by seeds would be very helpful for me.

**Comment by anthonyraymond** [comment]
Yep it's kinda what i want to achieve. I'm alone on this project so far ^^, so i'll need some time. But it's definely the next feature to be added to joal.
